packages/manager/modules/bm-server-components/src/general-information/installation/ovh/server-installation-ovh.controller.js

```diff
@@ -373,6 +394,11 @@ export default class ServerInstallationOvhCtrl {
   loadStep1() {
     this.$scope.loader.loading = true;
 
+    const hasSomePersonalTemplates = this.Server.getPersonalTemplatesList().then((templateList) => {
```

Comment:
In case the API call returns 404 (deprecated API call removed because migration finished), function should returns false
So we don't have manager-ovh dependency in the personal template /me/installationTemplate API call removal